store/sdk/src/main/java/org/apache/carbondata/sdk/file/CarbonReaderBuilder.java

```diff
@@ -118,6 +122,16 @@ public CarbonReaderBuilder withHadoopConf(String key, String value) {
     return this;
   }
 
+  /**
+   * Configure Vector Reader for carbonReader.
+   *
+   */
+  @Deprecated
+  public CarbonReaderBuilder withVectorReader(boolean useVectorReader) {
+    this.useVectorReader = useVectorReader;
+    return this;
```

Comment:
As per the comment given earlier modify the method and set the flag to true when this method is invoked